core/domain/exp_domain_test.py

```diff
@@ -7980,12 +8109,12 @@ class HTMLMigrationUnitTests(test_utils.GenericTestBase):
       translations_mapping:
         content: {}
         default_outcome: {}
-states_schema_version: 32
+states_schema_version: 33
 tags: []
 title: Title
 """)
 
-    YAML_CONTENT_V37_WITH_IMAGE_CAPTION = ("""author_notes: ''
```

Comment:
Ditto

Reply:
@rohitkatlaa ptal!

Reply:
@kevinlee12 I have removed the duplicate yaml content which I have mentioned in the previous conversation.